packages/dcos-checks/extra/checks/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
We can leave this file out since we have a top-level license in the repo that should apply to everything in it. Looks like adminrouter also provides a license file but that doesn't seem right. It's the same license (ASL 2.0) and if they conflicted, that'd be a problem.